bridge.go

```diff
@@ -86,6 +86,21 @@ func cmdAdd(args *skel.CmdArgs) error {
 		logrus.Infof("container already has interface: %v, no worries", args.IfName)
 	}
```

Comment:
The diff of this file looks confusing but the code change is fairly simple. You can get a better feel if you look at the two commits separately.